firecracker-microvm/firecracker-containerd
# Update for compatibility with Go Firecracker SDK interface changes

Configuration and execution of the firecracker process and VM changed a bit recently. This change updates the runtime shim to support the newer abstractions.

Manual testing:
`make clean all && sudo make install`

runtime changes tested with `ctr run` with the following images:
```
docker.io/library/busybox:latest (including `--tty`)
docker.io/library/debian:stretch-slim (including `--tty`)
docker.io/library/hello-world:latest
```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
